src/commons.rs

```diff
@@ -20,11 +20,14 @@ use std::fmt;
 use std::sync::Arc;
 use tokio::sync::Mutex;
 
+pub type Seed = Signature;
+pub type Hash = [u8; 32];
```

Comment:
Any reason to not use this type in `RoundUpdate` as you did for `Seed`?

Reply:
And in all other fields that mean `Hash`

Reply:
That could be addressed with another PR.  Seed was updated because it used to be with wrong size making both impl incompatible.